docs/jitserver_tuning.md

```diff
@@ -44,9 +44,12 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). The default is to use a nameless cache.
 
+ By default, the amount of memory that an AOT cache can consume at the server is not limited. The number of caches that a JITServer can hold is also not limited.
+
+ You can limit the amount of memory used by the JITServer AOT cache by using the [`-XX:JITServerAOTmx=<size>`](xxjitserveraotmx.md) option. Once the cache size limit is reached, new clients will not be able to create new AOT cache instances.
```

Comment:
Should also say that new compiled methods cannot be added to existing AOT cache instances.